gdb-server/src/bin.rs

```diff
@@ -15,11 +18,17 @@ struct Opt {
     )]
     chip_description_path: Option<String>,
     #[structopt(
-        name = "reset-halt",
-        long = "reset-halt",
-        help = "Use this flag to reset and halt (instead of just a reset) the attached core after flashing the target."
+        name = "halt",
+        long = "halt",
+        help = "Use this flag to halt core after attaching."
     )]
-    reset_halt: bool,
+    halt: bool,
```

Comment:
I'm not sure GDB even works properly if this flag is not set, because of the issue with reading the register of the core when it's not halted. Wouldn't it make sense to always halt when attaching?

Reply:
I am not sure what sideeffects a halt has (running timer that might not halt) but I guess this is not avoidable :/

Reply:
I don't think GDB can do anything when the core is running anyway, I don't think that we loose anything if we always halt the core.